docker_base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This is a base container we use. It has pgsql, redis and Python 3.5,
+# plus a user named qabel and locales.
+
+FROM ubuntu:16.04
```

Comment:
Also see the comment below. The download is actually much smaller than the on-disk size, so it's probably not an issue.